cli/cmd/cat.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/checkpoint"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/modules"
+	"github.com/tarantool/tt/cli/util"
+	"github.com/tarantool/tt/cli/version"
+)
+
+// catFlags contains flags for cat command.
+// Initialized with default values at creation.
+var catFlags = checkpoint.CmdOpts{
+	From:       0,
+	To:         18446744073709551615,
```

Comment:
What is `18446744073709551615`? Leave some comment.

Reply:
Fixed. Now it looks like `To: math.MaxUint64` inside structure with a comment in front of structure